content/news/041/index.md

```diff
@@ -77,6 +77,29 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library Updates
 
+### [Inox2d](https://github.com/Inochi2D/inox2d/)
+
+![The Japanese character for "inochi" set into a sprocket](https://raw.githubusercontent.com/Inochi2D/inox2d/main/inox2d_logo.svg)
```

Comment:
How about using the repo's social preview? I think it'll render better.

```suggestion
![The Japanese character for "inochi" set into a sprocket](https://repository-images.githubusercontent.com/566317952/7b02910a-a581-407e-80a9-fcedd26bc8b9)
```

Reply:
Kindaa, SVG is rendered client-side.

Reply:
I mean yeah that's true, I was more thinking of the background.

Reply:
@fianathedevgirl Oh, I missed the discussion about the image. Do you want it to be changed? The original svg looks quite fine to me